takari-builder/src/main/java/io/takari/builder/resolver/DependencyResolver.java

```diff
@@ -1,16 +1,17 @@
-package io.takari.builder.internal;
+package io.takari.builder.resolver;
```

Comment:
why non-internal package? do you expect `@Builders` to access classes from this package?

Reply:
I was thinking of future versions of this API, where takari-builder is pure API, and maven/other implementations are in external modules. I would want those to still be able to have access to these types of classes. In a way there would be 2 APIs, one intended for Builder implementors, and another intended for specific runtime implementors (such as the maven implementation). This is probably overthinking it for now, though...